src/main/webapp/WEB-INF/web.xml

```diff
@@ -4,7 +4,7 @@
          xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd"
          version="3.1">
 
-    <display-name>My Website</display-name>
```

Comment:
Nah it doesnt display anywhere, but I image if we do some weird internal Tomcat stuff the name might popup out of nowhere so I figured I'd give it a better name